src/game_api/script/lua_backend.hpp

```diff
@@ -94,6 +94,7 @@ enum class ON
     PRE_LOAD_SCREEN,
     POST_LOAD_SCREEN,
     DEATH_MESSAGE,
+    EVERYFRAME,
```

Comment:
I'd still rather call this event UPDATE (and then PRE_UPDATE for the other thing I asked)

Reply:
I still think it should have "frame" in the name, to signify that it's executed with the game framerate

Reply:
Well I think "every frame" is just confusing if you're playing at 300fps. A comment should make it clear enough that it runs at 60fps, after every state update

Reply:
But you can't play at 300fps, unless i don't know about some special tech?

you can play on monitor with some high refresh rate, but that's not the same

Reply:
My game runs at 300fps and I see 144 of those frames on my monitor. Calling something 60fps "every frame" is neither of those things. If you really have to have FRAME in the name, call it LOGICFRAME or something.

Reply:
With that logic ON.FRAME is very bad name and GAMEFRAME even worst

Reply:
Ok lets make everything bad cause other things are bad

Reply:
I'm not trying to argue that this event is not a "frame", just like those other things are not "frames", I'm just saying this ain't fucking EVERY FRAME